.pre-commit-config.yaml

```diff
@@ -1,47 +1,104 @@
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
 repos:
-  - repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v4.5.0
+  - repo: local
     hooks:
-      - id: check-yaml
+      - id: ruff-check
+        name: 🐶 Ruff Linter
+        language: system
+        types: [python]
+        entry: scripts/run-in-env.sh ruff check --fix
+        exclude: ^aiohue/v1/
```

Comment:
I don't understand why the ruff settings in pyproject.toml aren't applied for pre-commit. It would be better if we could remove this, but added it for now to make CI pass.